src/components/Faucet/index.tsx

```diff
@@ -99,59 +107,91 @@ export default function FaucetPage({
     signMessageSuccess,
     signMessageData,
     signMessageError,
-    signMessageLoading
+    signMessageLoading,
+    faucetTokenRequest
   ])
 
   useEffect(() => {
-    const fetchAddress = async () => {
+    const fetchAddressAndNetwork = async () => {
       try {
         const provider = new ethers.providers.Web3Provider(window.ethereum)
         const signer = provider.getSigner()
         const address = await signer.getAddress()
```

Comment:
suggestion: we can utilize wagmi react hooks here to be consistent with the repository

see [useAccount](https://wagmi.sh/react/api/hooks/useAccount) / [useNetwork](https://1.x.wagmi.sh/react/hooks/useNetwork)

Reply:
This will be addressed in a subsequent pull request.